primitives/npos-elections/src/lib.rs

```diff
@@ -217,6 +217,13 @@ impl sp_std::cmp::PartialOrd for ElectionScore {
 	}
 }
 
+/// Utility struct to group parameters for the balancing algorithm.
+#[derive(Clone)]
```

Comment:
can probably go into the balancing.rs file and get re-exported here. 

Also harmless to derive Copy, but up to you.

Reply:
Added